### PR TITLE
Generalize textures used in Tinkerbots worlds 

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Add DarkParquetry.proto and OldPlywood.proto
    - [x] Create icons
- [x] Use them in the tinkerbots worlds
- [x] *Fix subdivision in icon creator*
- [x] Log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
👎 Should be jpg.

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
👍

File: change_logs/ChangeLog.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
 <li>BallJoint nodes can now be controlled using RotationalMotor and Brake nodes and their angles can be measured using PositionSensor nodes.</li>
 <li>Extern robot controllers can be launched from outside of Webots.</li>
 <li>Added the following PROTO objects: 'IntermodalContainer', 'IntermodalOfficeContainer', 'Radiator', 'WallPlug', 'FireExtinguisher', 'CapScrew', 'ElectricalPlug', 'EyeScrew', 'ScrewHole', 'Washer', 'DoubleFluorescentLamp' and 'OfficeTelephone'.</li>
-<li>Added the following PROTO appearances: 'CorrodedMetal', 'CorrugatedMetal', 'DamascusSteel', 'ElectricConduit', 'FlexibleAluminiumDuct', 'GalvanizedMetal', 'HammeredCopper', 'OsbWood', 'Plaster', 'RustyMetal', 'ScrewThread' and 'WireFence'.</li>
+<li>Added the following PROTO appearances: 'CorrodedMetal', 'CorrugatedMetal', 'DamascusSteel', 'DarkParquetry', 'ElectricConduit', 'FlexibleAluminiumDuct', 'GalvanizedMetal', 'HammeredCopper', 'OldPywood', 'OsbWood', 'Plaster', 'RustyMetal', 'ScrewThread' and 'WireFence'.</li>
```
Comment (DavidMansolino) [inline change_logs/ChangeLog.html]:
```suggestion
<li>Added the following PROTO appearances: 'CorrodedMetal', 'CorrugatedMetal', 'DamascusSteel', 'DarkParquetry', 'ElectricConduit', 'FlexibleAluminiumDuct', 'GalvanizedMetal', 'HammeredCopper', 'OldPlywood', 'OsbWood', 'Plaster', 'RustyMetal', 'ScrewThread' and 'WireFence'.</li>
```

**Pull request merged**